report_worker/registry.py

```
@@ -85,53 +89,99 @@ def generate(self, job_id: str, organization_id: str, parameters: Dict[str, Any]
 """
         return markdown_content
 
-class ConversationAnalyticsGenerator(ReportGeneratorBase):
-    """Generate analytics reports from conversation data"""
-    
+
+class BrandAnalysisReportGenerator(ReportGeneratorBase):
+    """Generate a brand analysis report"""
+
+    def generate(self, job_id: str, organization_id: str, parameters: Dict[str, Any]) -> str:
+        """Generate a brand analysis report"""
+        # Extract brand_focus and industry_context from parameters
+        brand_focus = parameters.get(
+            'brand_focus', "Loctite's consumer and construction adhesives business")
+        industry_context = parameters.get(
+            'industry_context', "Consumer & Professional Adhesives and Sealants")
```

brand focus and industry context are extremely important. If they’re missing, lets throw an error don’t use a fallback